hw4/03-form.js

```diff
@@ -9,8 +9,25 @@ app.use(express.static('public'));
 app.use(express.urlencoded({ extended: false }));
 
 // POST request
-app.post('', (req, res) => {
+app.post('/submit', (req, res) => {
   // Add your code here
+  const output = (key, value) => {
+    const answer = value || 'n/a';
+    return `${key}: ${answer}<br />`;
+  }
+
+  res
+    .status(200)
+    .set({ 'Content-Type': 'text/html' });
+
+  let body = ''
+  for (let item in req.body) {
+    body += output(item, req.body[item])
+  }
+  res.send(body)
+
+  res.end()
+
 });
 
```

Comment:
Nice approach to store all data in chunks and then iterate over it to extract key and values.